modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -38,10 +38,27 @@ If you disable the Inbound Traffic tab, changes can take several minutes to comp
 . In the *Resource Allocation* section:
 +
 The inbound load balancer runs on each controller node configured in Runtime Fabric. Refer to xref:manage-index.adoc[Manage Runtime Fabric] for detailed information.
+
+.. Select a mode.
+
+*** *Shared Mode* is the default setting if no dedicated load balancer node is added in Runtime Fabric. In this mode, the internal load balancer runs on all controller nodes with the amount of CPU and memory specified.
+
+*** *Dedicated Mode* specifies that the entire node is available, so you cannot specify the amount of cores and memory. If you are setting up a Runtime Fabric with dedicated nodes, include only the dedicated node IP addresses. If *Dedicated Mode* is disabled, then all nodes are already in sharing mode, which prevents them from being dedicated. 
 +
-*Shared Mode* is the default setting if no dedicated load balancer node is added in Runtime Fabric. In this mode, the internal load balancer runs on all controller nodes with the amount of CPU and memory specified.
+The number of replicas for inbound traffic is set to the number of controller nodes for *Shared Mode* or the number of ingress nodes for *Dedicated Mode*. You must include all IP addresses of the controller nodes or the dedicated ingress nodes in your load balancer or DNS resolution. 
 +
-*Dedicated Mode* specifies that the entire node is available, so you cannot specify the amount of cores and memory. If *Dedicated Mode* is disabled, then all nodes are already in sharing mode, which prevents them from being dedicated.
+[WARNING]
+====
+When switching between shared and dedicated mode:
+
+* All traffic is lost if the shared or dedicated IP addresses are not included in the load balancer or DNS resolution 
+prior to making the change.
+
+* If you change from *Shared Mode* to *Dedicated Mode*, you must include the dedicated ingress node IPs in your load balancer in place of the controller node IPs. It might be necessary to temporarily include both controller and dedicated ingress nodes until the switch is completed to properly route input requests.
```

Comment:
* ..in your external load balancer..
* "It might be necessary to temporarily include" -> "MuleSoft recommends temporarily including"
* standardize the name of the nodes.
* .. to properly route inbound requests (or ingress).

Reply:
Updated - thank you!